client/src/Views/Home/Home.js

```diff
@@ -50,6 +51,8 @@ const Home = props => {
         path={`*/email-auth-results-dialog`}
         component={GmailAuthResultDialog}
       />
+      <Route path={`*/templates/load`} component={TemplateEditor} />
+      <Route path={`*/templates/create`} component={TemplateEditor} />
```

Comment:
I don't know if these routes are being used right?

Reply:
Right, I was initially pulling the editor up as a route vs a modal